lib/server.coffee

```diff
@@ -169,6 +167,7 @@ WebApp.connectHandlers.use (req, res, next) ->
 							# Extract JSON stringified phantomJS response after removing other potential Phantom logging messages. This regex extracts just the JSON.
 							try
 								output = JSON.parse stdout.replace /^(?!(\{.*\})$)(.*)|\r\n/gim, ''
```

Comment:
I've tested on one of our production sites (lots of ads on each page) - no issues was found